superset-frontend/spec/javascripts/utils/safeStringify_spec.ts

```diff
@@ -19,7 +19,7 @@
 import { safeStringify } from '../../../src/utils/safeStringify';
 
 class Noise {
-  public next: Noise;
+  public next!: Noise;
```

Comment:
Looks like this should actually be a `?` since in the test, Noise types don't always have a next set

Reply:
Thanks! I was new to typescript and was not really about the difference between "?:" and "!:". Just followed one of the suggested fixes from tslint (which is now deprecated I guess).